.gitignore

```diff
@@ -3,7 +3,12 @@ node_modules
 package-lock.json
 .env
 client/dist/bundle.js
-
+database/helpers/reviews.csv
```

Comment:
if you want you can add *.csv so that all csvs are ignored

Reply:
awesome! I added it to the .gitignore file